### PR TITLE
{bio}[intel/2018a] Clustal-Omega 1.2.4 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.14
See https://gist.github.com/a41e20f855740eb3d75a8d616fbb905b for a full test report.

File: easybuild/easyconfigs/a/argtable/argtable-2.13-intel-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ command line options with a minimum of fuss. """
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%s%s.tar.gz' % (name, version.replace('.', '-'))]
+checksums = ['8f77e8a7ced5301af6e22f47302fdbc3b1ff41f2b83c43c77ae5ca041771ddbf']
+
+sanity_check_paths = {
+    'files': ['lib/libargtable2.so', 'lib/libargtable2.la'],
```
Comment (verdurin) [inline easybuild/easyconfigs/a/argtable/argtable-2.13-intel-2018a.eb]: @hajgato Please use the shared library macro here

Comment (hajgato) [reply easybuild/easyconfigs/a/argtable/argtable-2.13-intel-2018a.eb]: @verdurin Done!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e3ba8f4b16f5bcfcd69f28c061b57096 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2400.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/61d6d18216f887ee30b8d66c483d290a for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**